src/wiki/plugins/images/models.py

```diff
@@ -108,7 +108,11 @@ def on_image_revision_delete(instance, *args, **kwargs):
     if not instance.image:
         return
     # Remove image file
-    path = instance.image.path.split("/")[:-1]
+    import storages
```

Comment:
This module doesn't exist (it's optional) -- the reason it passes tests is actually because this isn't covered by any tests :/

I think you should change this to a `try ... except` block. Catch `NotImplemented`  and then just `return` when it doesn't exist, because then there's no path to delete.

Reply:
of course, will do. Could you give me a hint on how I should write tests? Id' be happy to learn.